scripts/notify-suppliers-whether-application-made-for-framework.py

```diff
@@ -56,17 +59,15 @@
     context_data = context_helper.get_users_personalisations()
     error_count = 0
     for user_email, personalisation in context_data.items():
-        logger.info(user_email)
-        template_id = (NOTIFY_TEMPLATE_APPLICATION_MADE
-                       if personalisation['applied']
-                       else NOTIFY_TEMPLATE_NO_APPLICATION)
+        template_key = 'application_made' if personalisation['applied'] else 'application_not_made'
+
         if DRY_RUN:
-            logger.info("[Dry Run] Sending email {} to {}".format(template_id, user_email))
+            logger.info("[Dry Run] Sending {} email to {}".format(template_key, hash_string(user_email)))
```

Comment:
```suggestion
            logger.info("[Dry Run] Sending '{}' email to {}".format(template_key, hash_string(user_email)))
```

Just to make it clear that application_made is a key word